cpp/include/milvus-storage/format/parquet/file_reader.h

```diff
@@ -22,6 +22,20 @@ namespace milvus_storage {
 
 class FileRowGroupReader {
```

Comment:
question here: `FileRowGroupReader` is called by Milvus. Changing this API would require the changes to Milvus,  Is that expected?